packages/main/src/pages/login/Login.module.css

```diff
@@ -1,5 +1,9 @@
+#authPageWrapper * {
+  font-family: "Lato", sans-serif !important;
+}
+
 #authPageWrapper {
-  font-family: "Lato", sans-serif;
+  /* font-family: "Lato", sans-serif; */

```

Comment:
If the code is not being used, you should do away with it entirely instead fo commenting it out

Reply:
Also, ensure you add screenshots of the component before and after your changes